src/extensions/default/bramble/lib/ConsoleInterfaceManager.js

```diff
@@ -42,10 +42,11 @@ define(function (require, exports, module) {
         wasClosedByUser = false,
         unreadCount = 0,
         consoleEl = null,
+        linesEl = null,        
         maxLogs = 30;
 
     function clear() {
-        consoleEl.find(".log-entry").remove();
+        linesEl.find("div").remove();
```

Comment:
Why this change?

Reply:
I changed this to find all the divs within the lines element (which includes both log-entry and the line numbers elements) and remove them all when clear is pressed. If I kept the original line I would've had to add another line to find the line number elements and remove them separately.